index.js

```diff
@@ -43,3 +44,31 @@ function calenderTutorPopupOpen(){
     document.getElementById("day_popup").style.display = "flex";
     console.log("here");
 }
+
+function verifyInfo(fname, lname, emailEntered, inputtxt){
+    if(fname.value == "" || lname.value == ""){
+        alert("Name fields must not be empty!")
+    }else{
+        var emails = emailEntered.value;
+        
+        if(emails.length != 20){
+            alert("Please enter valid UB email length: \nUB Email length: 20 \nYour email length: " + emails.length)
+        }else{
+            var address = emails.slice(9,20); 
+            var emailName = emails.split("@buffalo.edu", 1);
+             
+            if(emailName[0].length != 8 || address != "buffalo.edu"){
+                alert("Please enter valid UB email! \nUB Email: example9@buffalo.edu\n \texamples@buffalo.edu\nYour email: " + emailName)
+            }else{
+                var decimal=  /^(?=.*\d)(?=.*[a-z])(?=.*[A-Z])(?=.*[^a-zA-Z0-9])(?!.*\s).{8,15}$/;
+                if(inputtxt.value.match(decimal)){ 
+                    alert('All information is valid')
+                    window.open("login.html", "self");
+                }
+                else{ 
+                    alert('Password must be 8 to 15 characters long and have at least 1 uppercase and lowercase letter, 1 number, and 1 special character.' )
+                }
+            }  
+        } 
+    }
+}
```

Comment:
please be consistent with your variable names. you mix up camel case and then sometimes don't use the camel case which just results into bad code. it's hard to read and follow. 

Also, your function names need to be more descriptive of what they are doing. currently, verifyInfo is not descriptive enough and is confusing. what info are you verifying? if you feel you are still struggling to come up with good function names because it takes time to get a hang of it, then add comments that describe what your function is supposed to do.